tests/base.py

```diff
@@ -251,8 +252,7 @@ def setUp(self):
         self.app = TestAppJSONAPI(api_django_app)
 
 
-@override_settings(DEBUG_PROPAGATE_EXCEPTIONS=True)
```

Comment:
@Ghalko, plz move this setting back to the decorator only so it only applies to the test cases and ping me with the pr (against 0.67.0) so i can merge, thx :octocat: